src/member/member.service.ts

```diff
@@ -352,4 +354,91 @@ export class MemberService {
   async getMemberTasks(memberId: string): Promise<Array<MemberTask>> {
     return this.memberInfra.getMemberTasks(memberId, this.entityManager);
   }
+
+  async timedMemberInfraFunction(name, memberInfraFunction, appId) {
+    const formattedTime = dayjs().format('YYYY-MM-DD HH:mm:ss.SSS');
+    const label = `${appId} - ${name} - ${formattedTime}`;
+    console.time(label);
+    const result = await memberInfraFunction();
+    console.timeEnd(label);
```

Comment:
remove console.log

Reply:
thanks for remind , as I discussed with you, since there is often a performance bottleneck here, I would like to be able to add execution time to the log
i have added more information to console.log 
`Execution Log - App ID: ${appId} | Function: ${name} | Timestamp: ${formattedTimestamp} | Execution Time: ${executionTime.toFixed(3)} ms`